### PR TITLE
Cairo 0.10.0 support

### PR DESCRIPTION
**Pull request by Solpatium**

### CODE REVIEW

File: starknet_py/net/gateway_client.py [hunk 1]
```diff
@@ -211,35 +213,32 @@ async def estimate_fee(
...
     async def call_contract(
         self,
-        invoke_tx: InvokeFunction,
+        invoke_tx: Union[InvokeFunction, Call],
         block_hash: Optional[Union[Hash, Tag]] = None,
         block_number: Optional[Union[int, Tag]] = None,
     ) -> List[int]:
-        """
-        Call the contract with given instance of InvokeTransaction
+        if isinstance(invoke_tx, InvokeFunction):
```
Comment (Solpatium) [inline starknet_py/net/gateway_client.py:228]: We shouldn't remove the docstring from here.

File: starknet_py/net/gateway_client.py [hunk 2]
```diff
@@ -363,6 +362,26 @@ async def get_code(
 
         return ContractCodeSchema().load(res, unknown=EXCLUDE)
 
+    async def get_contract_nonce(
+        self,
+        contract_address: int,
+        block_hash: Optional[Union[Hash, Tag]] = None,
+        block_number: Optional[Union[int, Tag]] = None,
+    ) -> int:
+        block_identifier = get_block_identifier(
```
Comment (Solpatium) [inline starknet_py/net/gateway_client.py:371]: We're missing a docstring here.

**Review by Radinyn — APPROVED** [review]
LGTM!

**Comment by jorneycr** [comment]
Why are you delete it?

**Comment by Ayoubull** [comment]
how

**Comment by zain4081** [comment]
awesomm

**Pull request merged**